src/main/java/eu/dissco/dataexporter/controller/DataExporterController.java

```diff
@@ -1,8 +1,56 @@
 package eu.dissco.dataexporter.controller;
 
+import eu.dissco.dataexporter.exception.ForbiddenException;
+import eu.dissco.dataexporter.exception.InvalidRequestException;
+import eu.dissco.dataexporter.schema.ExportJobRequest;
+import eu.dissco.dataexporter.service.DataExporterService;
+import java.util.UUID;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.oauth2.jwt.Jwt;
 import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 
 @Controller
+@RequestMapping("")
+@Slf4j
+@RequiredArgsConstructor
 public class DataExporterController {
 
+  private final DataExporterService service;
+
+  @PostMapping("/schedule")
+  public ResponseEntity<Void> scheduleJob(Authentication authentication,
+      @RequestBody ExportJobRequest request)
```

Comment:
I feel like I am missing ExportJobRequest, was it already pushed in an earlier commit?

Reply:
It's a jsonschema2pojo object

Reply:
:+1: